docs/excel/custom-functions-best-practices.md

```diff
@@ -0,0 +1,184 @@
+---
+ms.date: 09/18/2018
+description: Learn best practices and recommended patterns for Excel custom functions.
+title: Custom Functions' best practices
+---
+
+# Custom Functions' best practices
+
+This article covers some recommended patterns and solutions to common use cases with Excel custom functions.
+
+## Authentication
+
+Non-custom functions add-ins which have web views (i.e. display information in the taskpane) accomplish authentication via `Office.context.displayDialogAsync`. Custom functions differ from this approach. Because custom functions use a different runtime, they do not have access to the Office.context methods. Instead, they use methods available on the `OfficeRuntime` object, such as `OfficeRuntime.DisplayWebDialog()` and `OfficeRuntime.AsyncStorage.setItem(key, value)`.
```

Comment:
Can we just delete the sentence "Custom functions differ from this approach."? It doesn't seem needed.